spinalcordtoolbox/resample/nipy_resample.py

```diff
@@ -0,0 +1,176 @@
+#########################################################################################
+#
+# Resample data.
```

Comment:
Could indicate that this is using nipy here.

Reply:
Added a comment.